Index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE >
+<html lang="en">
+
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <title>Protfolio Mobile-setup</title>
+
+  <!-- Load Styles' file -->
+  <link href="styles.css" rel="stylesheet">
+
+  <!-- Load Fonts -->
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header_section">
+
+    <img src="Sources/Union.png" alt="Unicon vector" id="unicon">
+    <span id="name">Omar Farouq</span>
+
+  </header>
+  <main>
+
+    
+
+    <section class="headline_section">
+
+        <img src="Sources/Group.png" alt="illustrations' vector"
+         id="bkg_illustrations">
+        
+        <div id="primary_text">
+          Hey There. I’m Omar
+        </div>
+
+        <div id="supporting_text">
+            I’am a Software Developer
+        </div>
+
+        <div id="description">
+            I can help you build a product , feature 
+            or website Look through some of my work
+            and experience! If you like what you see
+            and have a project you need coded, 
+            don’t hestiate to contact me.
+        </div>
+
+      <div class="social_media">
+        <img src="Sources/Github.png" alt="Github icon">
+        <img src="Sources/Linkedin.png" alt="Linkedin icon">
+        <img src="Sources/Angelist.png" alt="Angelist icon">
+        <img src="Sources/Twitter.png" alt="Twitter icon">
+        <img src="Sources/Medium.png" alt="Medium icon">
+      </div>
```

Comment:
Please consider wrapping the social media icons in a `ul` tag and each `img` tag should also be wrapped in a `li` tag with a link to the respective social media page. 

A link per icon will let you connect to your social networks and other users can use them to connect with you online.